andela/lotus-ah
# #160451603 Users should be able to tag their articles

#### What does this PR do?
- Create tag
- Get all tag
- Append tag to articles
   
#### Description of Task to be completed?
  - Implemented create tag
  - Implemented get all tag
  - Implemented tag an article
  - Implemented untag an article
  - Get all Articles with a common tag

#### How should this be manually tested?
- Start the app locally on port `5000`
- Create a tag via postman with the structure below
```js
{ 
  name: 'react',
}
```
- Make a `POST` request to `/api/tags/`
- You should get a response with the structure below:
```js
{
  status: 'success',
  message: 'Tags added'
}
```

#### Any background context you want to provide?
NA

#### What are the relevant pivotal tracker stories?
[#160451603](https://www.pivotaltracker.com/story/show/160451603 "Users should be able to tag their articles")
